types/reactstrap/index.d.ts

```diff
@@ -81,7 +86,6 @@ export { default as TabContent,           TabContentProps                 } from
 export { default as Table,                TableProps                      } from './lib/Table';
 export { default as TabPane,              TabPaneProps                    } from './lib/TabPane';
 export { default as Tag,                  TagProps                        } from './lib/Tag';
-export { default as TetherContent,        TetherContentProps              } from './lib/TetherContent';
```

Comment:
removal of tether is a breaking change in [v 5-alpha](https://github.com/reactstrap/reactstrap/blob/b75a7e554147f52f45674733b3e01f0fbf39dd3f/CHANGELOG.md#breaking-changes).

First, the version should be updated to reflect this, see https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version since it is a major version.

Second, consider nod making the breaking change untill a stable release is available.

Reply:
sorry, my bad.